### PR TITLE
IS as a KM doc

### PR DESCRIPTION
**Pull request by isharac**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by tharindu1st — APPROVED** [review]

File: en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md [hunk 1]
```diff
@@ -1,77 +1,64 @@
...
+## Step 3 - Install and configure the databases
 
 You can create the required databases for the API-M deployment on a separate server and point to the databases from the respective nodes.
 
 The following diagram depicts how the databases are shared between WSO2 IS and WSO2 API-M.
 
-<a href="{{base_path}}/assets/img/setup-and-install/is-as-km-dbs.png" ><img src="{{base_path}}/assets/img/setup-and-install/is-as-km-dbs.png" alt="IS-AS-KM-DBS" title="IS-AS-KM-DBS" width="70%" /></a>
-
--   **WSO2AM_DB** - This database stores the identity data and API-related data and it includes OAuth tokens and keys. When serving key-validation requests, the Key Manager accesses the `WSO2AM_DB` , validates whether there are subscriptions made by the particular key. 
+<a href="{{base_path}}/assets/img/setup-and-install/is-as-km-dbs.png" ><img src="{{base_path}}/assets/img/setup-and-install/is-as-km-dbs.png" alt="IS-AS-KM-DBS" title="IS-AS-KM-DBS" width="90%" /></a>
```
Comment (Mariangela) [inline en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md:40]: The images states "Store". This needs to state "Developer Portal".

**Review by Mariangela — APPROVED** [review]

File: en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md (hunk 1 above)
Comment (isharac) [reply en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md:40]: This image is under reconstruction. Will add it once I receive it.

**Comment by Mariangela** [comment]
@isharac , the following changes need to be done in the mentioned images as there are some typos in the images as well.

**en/docs/assets/img/setup-and-install/dev-portal-connections.png**

Jms > JMS
OAuth2 application management > OAuth2 Application Management
Shared database > Shared Database
AM database > AM Database

**en/docs/assets/img/setup-and-install/gateway-connections.png**
Jms > JMS

**en/docs/assets/img/setup-and-install/key-manager-connections.png**
Jms > JMS
http > HTTP
failover > Failover
Shared database > Shared Database
AM database > AM Database
Block conditions > Block Conditions
token revocation > Token Revocation

**en/docs/assets/img/setup-and-install/publisher-connections.png**
API GAteway > API Gateway
Shared database > Shared Database
AM database > AM Database
Deploy API artifacts > Deploy API Artifacts
Jms > JMS

**en/docs/assets/img/setup-and-install/traffic-manager-connections.png**
controller events > Controller Events
Execution [lans > Execution Plans
NES > NFS
Shared database > Shared Database
AM database > AM Database

**Comment by Mariangela** [comment]
@isharac , once the changes are done, please send these changes to the master branch as well.

**Pull request merged**